CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   non-alpha-numeric passwords sent via stdin raised an error.
   [cyberark/conjur#2083](https://github.com/cyberark/conjur/issues/2083)
 
+### Changed
+- Updated return code on failure to encode batch secret response into JSON format.
```

Comment:
let's be more specific here -
> The batch secret retrieval endpoint now returns a `406 Not Acceptable` instead of a generic 500 error when a client requests a secret be returned with an incompatible encoding.

or something like that